libs/adapters/src/utils/setupErrorHandlers.ts

```diff
@@ -46,14 +46,22 @@ export const setupErrorHandlers = (app: Express) => {
         error: error.message,
       });
     } else {
-      log.error(error.message, error, reqContext);
-      res.status(500);
-      res.json({
-        status: error.status,
-        error:
-          error.message ||
-          'Server error, unknown error thrown. Please try again later.',
-      });
+      if (error?.status < 500) {
+        log.warn(error.message || '', error, reqContext);
```

Comment:
Can we add a placeholder message here for unknown error in the event we have just a status and not a message?